scripts/clean_obsolete_strings.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+
+'''
+This script is used to remove obsolete strings from all locales,
+and reformat files accordingly to the source content (en-US).
+'''
+
+from compare_locales.parser import getParser
+from compare_locales.serializer import serialize
+import os
+import subprocess
+import sys
+
+
+def extractFileList(repository_path):
+    '''
+        Extract the list of supported files. Store the relative path and ignore
+        specific folders for products we don't want to modify (e.g. SeaMonkey).
+    '''
+
+    supported_formats = [
```

Comment:
Should we include `.ini`?

Reply:
I'd give that a more thorough review on the output, but also, probably not many wins there.